umd-templates/web.js

```diff
@@ -6,6 +6,12 @@ if (!this.<%= namespace %>) (function(doc, ev) {
 
   // dispatch load event
   ev = doc.createEvent('HTMLEvents');
-  ev.initEvent('sentinel-load', false, false);
+  
+  // the initEvent may be deprecated in the future, instead we can use the Event constructor
+  ev.initEvent ? ev.initEvent('sentinel-load', false, false) : new Event('sentinel-load', {
```

Comment:
It looks like the `ev` variable created with `doc.createEvent()` isn't being replaced with the new `Event` instance. Also, `bubbles` and `cancelable` are both false by default so we can save some bytes by not passing arguments.

Reply:
Sorry, I think I missed some details, so I assumed that it will work like this: 
ev.initEvent ? ev.initEvent('sentinel-load', false, false) : (ev = new Event('sentinel-load'));